C sharp/Thorlabs CCS Spectrometers/CCS - Absorption Measurement/CCS - Absorption Measurement/Program.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System;
```

Comment:
The code looks great. We might need to make a few changes to method names to make them more clear. We should also update the readme to include a description of the new example